proto/one/common/model/model_input.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+import "google/protobuf/wrappers.proto";
+import "model_state.proto";
+import "model_parameter.proto";
+
+option csharp_namespace ="ONE.Models.CSharp";
+
+message ModelInput {
+    //Model 
+    google.protobuf.StringValue model = 1;
+    //Comment
+    google.protobuf.StringValue comment = 2;
+    //Length of forecast
+    google.protobuf.StringValue forecastDuration = 3;    
+    //Model state contains inputs and outputs that are each a colleciton of model parameters
+    ModelState state = 4;
```

Comment:
I don't have the confluence page in front of me but I don't think state would have outputs.  I think we would have 3 collections of modelParameters, state, inputs, outputs or maybe stateParameters, inputParameters, outputParameters